paywall/package.json

```diff
@@ -14,6 +14,7 @@
     "lint": "eslint --ext .tsx,.ts,.js src/",
     "svg-2-components": "./node_modules/@svgr/cli/bin/svgr --title-prop --no-dimensions --template src/components/interface/svg/template.js --no-dimensions -d src/components/interface/svg/ src/static/images/svg/",
     "build-checkout": "yarn build-unlock.1.0.js && yarn build-data-iframe.1.0.js",
+    "watch-checkout": "webpack --watch --config unlock.2.0.js.webpack.config.js",
```

Comment:
Pretty simple to have it automatically compile on change. The only downside is that this requires 2 terminal windows; otherwise the watcher blocks execution of the main paywall app.

Reply:
Would you mind updatting the README as well so we can keep track of that?